.github/workflows/image-build-and-draft-release.yaml

```diff
@@ -16,7 +16,6 @@ permissions:
 ## Define which docker arch to build for
 env:
   docker_platforms: "linux/amd64"
-  docker-org: blockstack
   latest_release: $(curl -s https://api.github.com/repos/${{ github.repository }}/releases/latest | jq -r .tag_name)
```

Comment:
Nice, I didn't actually know that you could assign env's dynamically like this 👍